packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/VSphereDetailsSection.tsx

```diff
@@ -32,20 +31,19 @@ export const VSphereDetailsSection: React.FC<DetailsSectionProps> = ({ data }) =
       <DetailsItem
         title={t('Type')}
         content={type}
-        moreInfoLink={HELP_LINK_HREF}
+        moreInfoLink={
+          'https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.4/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#adding-providers'
+        }
         helpContent={
-          <Text>{t(`Allowed values are openshift, ovirt, vsphere, and openstack.`)}</Text>
+          <Text>
+            {t(
+              'Specify the type of source provider. Allowed values are ovirt, vsphere, and openstack. This label is needed to verify the credentials are correct when the remote system is accessible and, for RHV, to retrieve the Manager CA certificate when a third-party certificate is specified.',
+            )}
+          </Text>
         }
         crumbs={['Provider', 'spec', 'type']}
       />
 
-      <DetailsItem
```

Comment:
if you remove an item you will need to add a placeholder, o/w the order of the items in this section will be changed